gateway/src/http_entrypoints.rs

```diff
@@ -1224,6 +1290,40 @@ async fn sp_rot_cfpa_get(
     Ok(HttpResponseOk(RotCfpa { base64_data, slot }))
 }
 
+/// Read the RoT boot state from a root of trust
```

Comment:
This gets called implicitly as part of the sp state. I had this exposed as part of an earlier draft and kept it around. Might be better to remove if it's not being used?

Reply:
I made a first pass attempt and it was kind of a mess because of backwards compatibility for checking where the RoT info lives. I do think it's the right approach and I may see about doing that again with more of the wicket changes.